### ISSUE TITLE
kinex.js local file reference to migrations

### ISSUE THREAD

**Issue by aaronduchateau** [Bikranshu/express-react-boilerplate#1]
Hi there, good work on this project. I was following your readme, and it seems as though going through the initial steps do not generate the /src file referenced in your kenex.js file. 

Am i missing something here, or should i modify that path to point to the migrations directory at /server/migrations?

**Comment by I-keep-trying** [comment]
Yeah, I don't understand either. I had the exact same problem. It's not anything obvious, that's for sure.

Could this be the file that's supposed to be there, in the /src directory? 
https://github.com/itsjives/knexfile_TEMPLATE/blob/master/knexfile.js

**Comment by PadharaHitesh** [comment]
Yeah, I understand Problem, just change the directory path line in knexfile.js

 **directory: './server/migrations'**
